demo/Sky3DDemo.tscn

```diff
@@ -1,60 +1,146 @@
+shader_parameter/_clouds_sky_tint_fade = 0.5
+shader_parameter/_clouds_intensity = 10.0
+shader_parameter/_clouds_size = 2.0
+shader_parameter/_clouds_uv = Vector2(0.16, 0.11)
+shader_parameter/_clouds_speed = 0.07
+shader_parameter/_clouds_direction = Vector2(0.25, 0.25)
+shader_parameter/_clouds_day_color = Color(0.823529, 0.87451, 1, 1)
+shader_parameter/_clouds_horizon_light_color = Color(0.98, 0.43, 0.15, 1)
+shader_parameter/_clouds_night_color = Color(0.090196, 0.094118, 0.129412, 1)
+shader_parameter/_clouds_texture = ExtResource("6_1g8sw")
+shader_parameter/_cumulus_clouds_coverage = 0.55
+shader_parameter/_cumulus_clouds_thickness = 0.0243
+shader_parameter/_cumulus_clouds_absorption = 2.0
+shader_parameter/_cumulus_clouds_noise_freq = 2.7
+shader_parameter/_cumulus_clouds_sky_tint_fade = 0.0
+shader_parameter/_cumulus_clouds_intensity = 1.0
+shader_parameter/_cumulus_clouds_size = 0.5
+shader_parameter/_cumulus_clouds_speed = 0.05
+shader_parameter/_cumulus_clouds_direction = Vector3(0.25, 0.1, 0.25)
+shader_parameter/_cumulus_clouds_texture = ExtResource("5_rt0jr")
+shader_parameter/_cumulus_clouds_day_color = Color(0.823529, 0.87451, 1, 1)
+shader_parameter/_cumulus_clouds_horizon_light_color = Color(0.98, 0.43, 0.15, 1)
+shader_parameter/_cumulus_clouds_night_color = Color(0.090196, 0.094118, 0.129412, 1)
+shader_parameter/_cumulus_clouds_partial_mie_phase = Vector3(0.957564, 1.04244, 0.412)
+shader_parameter/_cumulus_clouds_mie_intensity = 1.0
+
+[sub_resource type="Sky" id="Sky_0j4pb"]
+sky_material = SubResource("ShaderMaterial_v8wrf")
+
+[sub_resource type="Environment" id="Environment_1g8sw"]
+background_mode = 2
+sky = SubResource("Sky_0j4pb")
+ambient_light_source = 3
+ambient_light_color = Color(0.235156, 0.278907, 0.35, 1)
+ambient_light_sky_contribution = 0.7
+reflected_light_source = 2
+tonemap_mode = 3
+tonemap_white = 6.0
+
+[sub_resource type="CameraAttributesPractical" id="CameraAttributesPractical_d0ed4"]
 
 [node name="Main" type="Node3D"]
 
 [node name="PrimitivesCity" parent="." instance=ExtResource("2")]
 
 [node name="MainCamera" type="Camera3D" parent="."]
-transform = Transform3D(0.258819, -0.0505527, -0.964602, 0, 0.99863, -0.052336, 0.965926, 0.0135455, 0.258464, -7.677, 2.522, 3.686)
-environment = ExtResource("2_203ee")
-attributes = ExtResource("3_bgocq")
+transform = Transform3D(0.0795793, -0.05217, -0.995462, -2.58349e-10, 0.99863, -0.052336, 0.996829, 0.00416486, 0.0794703, -7.677, 2.522, 3.686)
 fov = 70.0
 far = 8192.0
 
 [node name="ReflectionProbe" type="ReflectionProbe" parent="."]
+visible = false
 update_mode = 1
 size = Vector3(2000, 2000, 2000)
 cull_mask = 4
 
 [node name="Sky3D" type="WorldEnvironment" parent="."]
-environment = ExtResource("2_203ee")
-camera_attributes = ExtResource("3_bgocq")
+environment = SubResource("Environment_1g8sw")
+camera_attributes = SubResource("CameraAttributesPractical_d0ed4")
 script = ExtResource("2_cejat")
-current_time = 6.3
```

Comment:
Let's keep the demo starting early in the morning, with the reflection probe enabled.